src/main/kotlin/lotto/model/WinningTicket.kt

```diff
@@ -0,0 +1,8 @@
+package lotto.model
+
+class WinningTicket(val winningNumbers: Numbers, val bonusNumber: Int = 1) {
+    init {
+        require(bonusNumber in Const.MIN..Const.MAX) { Const.NUM_RANGE }
+        require(!winningNumbers.contains(bonusNumber)) { Const.DISTINCT_NUM }
```

Comment:
```suggestion
        require(bonusNumber !in winningNumbers) { Const.DISTINCT_NUM }
```

It's more idiomatic Kotlin, so I'd recommend using it!

Reply:
Thank you for advice. I'm add operator fun contains(number: Int): Boolean in Numbers.kt. And after changed  "!winningNumbers.contains(bonusNumber" to "bonusNumber !in winningNumbers".
Now it looks more readable